### ISSUE TITLE
Disabled & readonly form elements' contrasts

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#566]
Contrasts aren't sufficient for `[disabled]` and `[readonly]` elements (which leads us to [ignore them from pa11y](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/8145088d68168d39dc58f849bac466e494918b35/build/.pa11yci.json#L10)).

I'd recommend setting them to `#666` on `#eee`.

That includes:

- [x] `:disabled` and `[disabled]`
- [x] `:read-only` and `[readonly]`
- [x] `::placeholder`

For all kind of form control (select, file input, etc.).

**Comment by ffoodd** [comment]
Since there's no WCAG reference to argue this topic, let's simply match the UI Kit for now. Meaning the only covered things as of now are disabled and readonly field values (missing from the UI Kit :innocent:).
